gulpfile.js

```diff
@@ -1,14 +1,34 @@
 var copy = require("copy"),
     clear = require("cli-clear"),
     del = require("del"),
+    exec = require("child_process").exec,
     gulp = require("gulp"),
-    gutil = require("gulp-util");
+    gutil = require("gulp-util"),
+    runSequence = require("run-sequence");
 
 gulp.task("clear", function(cb) {
```

Comment:
Is this required, btw? Haven't seen this used in other gulp configurations

Reply:
I wanted the window to clear automatically whenever I started a test run (I absolutely hate scanning the terminal window trying to find where the last run ended and where the current started) and I wanted it to behave the same way on windows/*nix and `cli-clear` was the only package that actually worked on all platforms. This way the test run always starts at the top of the window, on all platforms, predictability!